js/components/generate/GenerateForm.js

```diff
@@ -298,35 +338,86 @@ export default class GenerateForm extends React.Component {
                       className="tooltip-icon"
                       style={{ marginTop: "-0.3rem" }}>
                     </a>
-                    <Tooltip id="courses-tooltip" place="right" />
+                    <Tooltip id="category-tooltip" place="right" />
                   </div>
+                  <Field as="select" id="category" name="category">
+                    <option value="courses">Courses</option>
+                    <option value="programs">Programs</option>
+                  </Field>
 
-                  <Field
-                    id="courses"
-                    name="courses"
-                    type="text"
-                    placeholder="e.g., CSC207H1, CSC324H1"
-                  />
-                  <div className="error-container">
-                    <ErrorMessage
-                      className="error-message"
-                      name="courses"
-                      component="div"
-                    />
-                  </div>
+                  {values.category === "courses" && (
+                    <>
+                      <div className="title-container">
+                        <h1 id="header-title" className="section-title">
+                          Search Courses
+                        </h1>
+                        <a
+                          data-tooltip-id="courses-tooltip"
+                          data-tooltip-html="Generate the prerequisites for the given course(s).<br />
+                        Each course code must follow the format CSC108H1<br />
+                        (i.e. department + code + session)"
+                          className="tooltip-icon"
+                          style={{ marginTop: "-0.3rem" }}>
+                        </a>
+                        <Tooltip id="courses-tooltip" place="right" />
+                      </div>
+                      <Field
+                        id="courses"
+                        name="courses"
+                        type="text"
+                        placeholder="e.g., CSC207H1, CSC324H1"
+                      />
+                      <div className="error-container">
+                        <ErrorMessage
+                          className="error-message"
+                          name="courses"
+                          component="div"
+                        />
+                      </div>
+                    </>
+                  )}
+
+                  {values.category === "programs" && (
+                    <>
+                      <div className="title-container">
+                        <h1 id="header-title" className="section-title">
+                          Search Programs
+                        </h1>
+                        <a
+                          data-tooltip-id="programs-tooltip"
+                          data-tooltip-html="Generate the requirements for the given program(s).<br />
+                        Each program code must follow the format ASFOC1689B"
```

Comment:
Please replace the example with `ASMAJ1689`; it's a bit misleading to use the focus code, because the last character ("B") is not present in Specialist/Major/Minor codes.